torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -16,8 +16,9 @@ defmodule Torchx.NxDoctestTest do
   @temporarily_broken_doctests [
     # broadcast - shape mismatch in one test
     broadcast: 3,
-    # conjugate, real, imag - Torchx does not support complex numbers
+    # conjugate, phase, real, imag - Torchx does not support complex numbers
     conjugate: 1,
+    phase: 1,
```

Comment:
This shouldn't be skipped here

Reply:
Where should it be skipped then? `phase` depends on `real` and `imag` and thus won't work without them..